docker/dev/find_fastest_resources.sh

```diff
@@ -48,10 +49,11 @@ function find_fastest_url() {
         cur_speed=$(ping -c 4 -W 2 "$domain" | tail -1 \
                            | grep "/avg/" | awk '{print $4}'\
                            | cut -d '/' -f 2)
-        cur_speed=${cur_speed:-99999.9}
+        cur_speed=${cur_speed:-99999}
+        cur_speed=${cur_speed/.*/}
```

Comment:
Need a comment here explaining this Bash syntax.

Reply:
Ok